### PR TITLE
Add license for max license

### PR DESCRIPTION
**Pull request by popcornylu**
**PR checklist**
- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**Which issue(s) this PR fixes**:
sc-23683

**Reviewer note**:
Please remember to update the chart before using the test image
https://github.com/InfuseAI/primehub/pull/646

## Test cases
- [x] If maxUser is 1, we should not create user in admin > user
- [x] If maxUser is 1, we cannot send invitation
- [x] If maxUser is 1, we cannot create user from invitation
- [x] If maxUser is 1, show 'x/1' in system page
- [x] If maxUser is unlimited, we should not create user in admin > user
- [x] If maxUser is unlimited, we cannot send invitation
- [x] If maxUser is unlimited, we cannot  create user from invitation
- [x] If maxUser is unlimited, show 'x/∞' in system page
- [x] check if it does not affect ce case.

## Screenshots
Admin > Users
image

Home > Invite User
image

Invitation page
image

Admin > Systems
image

## Test image
- ~~sc23693-9abb144~~
- sc23693-a5eaba9

### CODE REVIEW

File: packages/client/src/admin/SystemSetting/__tests__/SystemSetting.spec.tsx [hunk 1]
```diff
@@ -30,7 +30,9 @@ function setup() {
               maxNode: 1000,
               maxModelDeploy: 1000,
               maxGroup: 9999,
+              maxUser: 9998,
```
Comment (neighborhood999) [inline packages/client/src/admin/SystemSetting/__tests__/SystemSetting.spec.tsx:33]: The number of *9998* is come from: `9999 - 1 (admin or myself)`?

File: packages/graphql-server/src/ee/resolvers/license.ts [hunk 2]
```diff
@@ -4,12 +4,19 @@ import { client as kubeClient } from '../../crdClient/crdClientImpl';
 import { Context } from '../../resolvers/interface';
 import { createConfig } from '../../config';
 
-const config = createConfig();
+let licenseOverride = {};
+
+// this is used for integratino test
```
Comment (neighborhood999) [inline packages/graphql-server/src/ee/resolvers/license.ts:9]: typo

File: packages/client/src/admin/SystemSetting/__tests__/SystemSetting.spec.tsx (hunk 1 above)
Comment (popcornylu) [reply packages/client/src/admin/SystemSetting/__tests__/SystemSetting.spec.tsx:33]: the number is just arbitrary number to test. TO user the different number form `maxGroup` is to make sure that i did not use the wrong variable.

File: packages/graphql-server/src/ee/resolvers/license.ts (hunk 2 above)
Comment (popcornylu) [reply packages/graphql-server/src/ee/resolvers/license.ts]: Fixed

**Review by neighborhood999 — APPROVED** [review]

**Pull request merged**